package.json

```diff
@@ -25,51 +41,54 @@
     "@types/recharts": "^1.8.19",
     "@types/styled-components": "^5.1.7",
     "@types/three": "^0.128.0",
+    "@vitejs/plugin-react": "^1.3.2",
```

Comment:
👀  some tooling

Reply:
yes, it's replacement for previous plugin